src/components/text/docs.mdx

```diff
@@ -0,0 +1,29 @@
+---
+componentName: Text
+---
+
+The `Text` component should be used for easily styling `p` elements to conform to the design system.
```

Comment:
Not critical, but maybe update the wording to indicate more than just `p`?

Reply:
Definitely! Forgot to update it when I added the support for extra elements. 😁

Reply:
Fixed in https://github.com/hashicorp/dev-portal/pull/88/commits/443f3604c283e3b7f54d68f952e9ce00b959507b. 😊